src/endpoints/branding.ts

```diff
@@ -14,18 +16,42 @@ class Branding extends BaseExtend {
   }
 
   /**
-   * Gets branding params for given merchant
+   * Gets branding details for given merchant.
    * @method getBranding
    * @async
-   * @param {string} clientId - The Client id
-   * @param {string} brandingId - The branding id or 'default'
+   * @param {string} clientId The Merchant's unique universal identifier (Merchant UUID).
+   * You can find it on the InPlayer's dashboard at the Account Details section as Account ID.
+   * @param {string | number} brandingId The id of created branding theme in InPlayer Dashboard.
+   * If it is not set the default branding details will be returned.
    * @example
    *     InPlayer.Branding
    *     .getBranding('eyJ0e-XAiOi-JKPEC-ENR5Y', '123')
    *     .then(data => console.log(data));
-   * @returns  {AxiosResponse<Brand>}
+   * @returns {AxiosResponse<BrandingDetails>} Contains the data:
+   * ```typescript
+   * {
+   *    id: number;
+   *    exists: boolean;
+   *    brand_name: string;
+   *    paywall_cover_photo: string;
+   *    paywall_brand_logo: string;
+   *    paywall_primary_color: string;
+   *    paywall_secondary_color: string;
+   *    paywall_buttons_bg_color: string;
+   *    paywall_buttons_text_color: string;
+   *    preview_buttons_bg_color: string;
+   *    preview_buttons_text_color: string;
+   *    preview_top_border: boolean;
+   *    inplayer_protected_label: boolean;
+   *    paywall_footer: string;
+   *    default: boolean;
+   *    template_id: number;
+   *    terms_url: string;
+   *    dark_mode: boolean;
+   * }
+   * ```
    */
-  async getBranding(clientId: string, brandingId: string | number) {
+  async getBranding(clientId: string, brandingId = 'default'): Promise<AxiosResponse<BrandingDetails>> {
```

Comment:
This implies that `brandingId` will always be a string, although apart from the `default` value, it's a number. Here's one way we can express that:

```suggestion
  async getBranding(clientId: string, brandingId: number | 'default' = 'default'): Promise<AxiosResponse<BrandingDetails>> {
```